### PR TITLE
fix hellogrpc/py examples by adding dependency

### PR DESCRIPTION
**Pull request by erain**
* All current Travis CI would fail because hellogrpc/py e2e test would
  fail. This PR fix this test, so that Travis CI works again.

* hellogprc/py server would produce a py_image which is failed to
  start because of missing a dependency:

``` bash
Traceback (most recent call last):
  File "/app/hello_grpc/py/server/server.binary.runfiles/__main__/hello_grpc/py/server/server.py", line 1, in <module>
    import grpc
  File "/app/hello_grpc/py/server/server.binary.runfiles/pypi__grpcio_1_6_0/grpc/__init__.py", line 22, in <module>
    from grpc._cython import cygrpc as _cygrpc
  File "src/python/grpcio/grpc/_cython/cygrpc.pyx", line 17, in init grpc._cython.cygrpc (src/python/grpcio/grpc/_cython/cygrpc.c:29041)
```

* Adding setuptools in requirements fixes the problem.

### CODE REVIEW

**Comment by erain** [comment]
/assign @nlopezgi @xingao267 

Could you help review this PR?

File: .travis.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ addons:
 
 before_install:
   # Log into GCR
-  - docker login -u _json_key -p "${GOOGLE_JSON_KEY}" https://us.gcr.io
+  - echo "${GOOGLE_JSON_KEY}" | docker login -u _json_key --password-stdin https://us.gcr.io
```
Comment (mattmoor) [inline .travis.yml]: This is not the issue.  Travis only provides a password to PRs from a branch on the repo.

You should revert this with:
```
git reset --hard HEAD~1
```

and reopen this PR from a branch on the bazelbuild repo.

Comment (erain) [reply .travis.yml]: I got stuck on this for a while and many thanks!!! 

So for external PRs, we need to first review / approve it, then create a PR on the bazelbuild repo?

**Pull request closed without merging**